skxray/core.py

```diff
@@ -1180,3 +1180,47 @@ def multi_tau_lags(multitau_levels, multitau_channels):
     lag_steps = np.append(lag_steps, np.array(lag))
 
     return tot_channels, lag_steps
+
+
+def circular_average(image, calibrated_center, thershold=0, nx=100,
+                     pixel_size=None):
+    """
+    Circular average(radial integration) of the intensity distribution of
+    the image data.
+
+    Parameters
+    ----------
+    image : array
+        input image
+
+    calibrated_center : tuple
+        The center in pixels-units (row, col)
+
+    thershold : float, optional
```

Comment:
This is compared against something that is always a integer.  This is setting the minimum number of pixels in the bins that will be included.  This doc string gives a very different impressoin.